.github/workflows/provider.yaml

```diff
@@ -64,6 +64,15 @@ jobs:
           astro dev run dags trigger lakeFS_workflow
           sleep 30
 
+      - name : Run DAG state check script
+        id : dag_status_id
+        run: |
+            python3 dag_status.py
+
+      - name : Check failed DAG satus
+        if: steps.dag_status_id.outputs.status !=0
+        run :  exit 1
```

Comment:
I'd like to get rid of this step, and I think that we can.  (Actually step `dag_status_id` doesn't define any outputs, so I am not sure how this step will work.)

Reply:
Removed this step